
===|Dev Logs :TSR_1556114364|===============================================

getpath invoked and returns :/home/automation/webapps/AutoBox/BDD/Core/../Logs/TSR_1556114364/Images/
FindElement invoked 
type invoked  :xpath
Find element for :/html/body/entry/login/div/div[2]/div/form/div[2]/input
<selenium.webdriver.remote.webelement.WebElement (session="747b652005e2634fb622a95053231174", element="0.3279734256185609-1")>
getpath invoked and returns :/home/automation/webapps/AutoBox/BDD/Core/../Logs/TSR_1556114364/Images/
FindElement invoked 
type invoked  :xpath



Find Element Failed :
Traceback (most recent call last):
  File "/home/automation/webapps/AutoBox/BDD/Core/SeleniumFunction.py", line 24, in FindElement
    element=driver.find_element_by_xpath(variable)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//i[@class='icon produi-icon-scenario']//parent::span//parent::button"}
  (Session info: headless chrome=73.0.3683.103)
  (Driver info: chromedriver=2.45.615279 (12b89733300bd268cff3b78fc76cb8f3a7cc44e5),platform=Linux 4.15.0-47-generic x86_64)





FindElement invoked 
type invoked  :xpath



Find Element Failed :
Traceback (most recent call last):
  File "/home/automation/webapps/AutoBox/BDD/Core/SeleniumFunction.py", line 24, in FindElement
    element=driver.find_element_by_xpath(variable)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//button[@label='Create']"}
  (Session info: headless chrome=73.0.3683.103)
  (Driver info: chromedriver=2.45.615279 (12b89733300bd268cff3b78fc76cb8f3a7cc44e5),platform=Linux 4.15.0-47-generic x86_64)





FindElement invoked 
type invoked  :xpath



Find Element Failed :
Traceback (most recent call last):
  File "/home/automation/webapps/AutoBox/BDD/Core/SeleniumFunction.py", line 24, in FindElement
    element=driver.find_element_by_xpath(variable)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//button[@title='Add Group']"}
  (Session info: headless chrome=73.0.3683.103)
  (Driver info: chromedriver=2.45.615279 (12b89733300bd268cff3b78fc76cb8f3a7cc44e5),platform=Linux 4.15.0-47-generic x86_64)





FindElement invoked 
type invoked  :xpath



Find Element Failed :
Traceback (most recent call last):
  File "/home/automation/webapps/AutoBox/BDD/Core/SeleniumFunction.py", line 24, in FindElement
    element=driver.find_element_by_xpath(variable)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//button[@icon='fa-caret-down']"}
  (Session info: headless chrome=73.0.3683.103)
  (Driver info: chromedriver=2.45.615279 (12b89733300bd268cff3b78fc76cb8f3a7cc44e5),platform=Linux 4.15.0-47-generic x86_64)




